docs/json_rpc.md

```diff
@@ -287,13 +292,26 @@ Example result:
         "SubmissionTime": 1642589195,
         "Status": "MINED",
         "FinalisationBlock": 84347
+    },
+    {
+        "ID": "5",
+        "Hash": null,
+        "Type": "TRANSFER",
+        "TransactionHash": "0xa428c43175e6156c933c436455a675c1e521e3841d9d44c6482ee26f861e49c1",
+        "SubmissionBlock": null,
+        "SubmissionTime": null,
+        "Status": "SUBMITTED",
+        "FinalisationBlock": null
     }
 ]
 ```
 
 ### `hubble_getBatchByHash(hash)`
 
-Returns batch information and list of included commitments in batch.
+Returns batch information and list of included commitments in batch. Commitment statuses:
```

Comment:
In this endpoint commitments don't have statuses. Maybe replace with something like:
`Returns mined or finalised batch information...`